app/config/collections.php

```diff
@@ -1,12 +1,15 @@
 <?php
 
-use Appwrite\Auth\Auth;
 use Utopia\Config\Config;
-use Utopia\Database\Database;
-use Utopia\Database\Helpers\ID;
 
-$providers = Config::getParam('oAuthProviders', []);
+$common = include __DIR__ . '/collections/common.php';
+$bucket = include __DIR__ . '/collections/bucket.php';
```

Comment:
```suggestion
$buckets = include __DIR__ . '/collections/buckets.php';
```
Lets also update `databases` and `projects` - both the file name and the variable names